tests/test_classifier/test_learning_curve.py

```diff
@@ -0,0 +1,150 @@
+            visualizer.poof()
+        except Exception as e:
+            self.fail("error during learning curve: {}".format(e))
+
+        self.assert_images_similar(visualizer)
+
+    def test_learning_curve_model_only(self):
+        """
+        Test learning curve with inputting model only. 
+        """
+
+        X = self.occupancy[[
+                "temperature", "relative_humidity", "light", "C02", "humidity"
+            ]]
+
+        y = self.occupancy['occupancy'].astype(int)
+
+        X = X.view((float, len(X.dtype.names)))
+        
+        try:
+            visualizer = LearningCurveVisualizer(LinearSVC())
+            visualizer.fit(X, y)
+            visualizer.poof()
+        except Exception as e:
+            self.fail("error during learning curve: {}".format(e))
+
+    def test_learning_curve_model_cv_only(self):
+        """
+        Test learning curve with inputting model and cv only.
+        """
+
+        X = self.occupancy[[
+                "temperature", "relative_humidity", "light", "C02", "humidity"
+            ]]
+
+        y = self.occupancy['occupancy'].astype(int)
+
+        X = X.view((float, len(X.dtype.names)))
+        
+        try:
+            visualizer = LearningCurveVisualizer(LinearSVC(),
+                cv=ShuffleSplit(n_splits=100, test_size=0.2, random_state=0))
+            visualizer.fit(X, y)
+            visualizer.poof()
+        except Exception as e:
+            self.fail("error during learning curve: {}".format(e))
+
+    def test_learning_curve_model_trainsize_cv_only(self):
+        """
+        Test learning curve with inputting model, training size, and cv only.
+        """
+
+        X = self.occupancy[[
+                "temperature", "relative_humidity", "light", "C02", "humidity"
+            ]]
+
+        y = self.occupancy['occupancy'].astype(int)
+
+        X = X.view((float, len(X.dtype.names)))
+        
+        try:
+            visualizer = LearningCurveVisualizer(LinearSVC(), 
+                train_sizes=np.linspace(.1, 1.0, 5),
+                cv=ShuffleSplit(n_splits=100, test_size=0.2, random_state=0))
+            visualizer.fit(X, y)
+            visualizer.poof()
+        except Exception as e:
+            self.fail("error during learning curve: {}".format(e))
+
+    def test_learning_curve_bad_trainsize(self):
+        """
+        Test learning curve with bad input for training size.
+        """
+
+        X = self.occupancy[[
+                "temperature", "relative_humidity", "light", "C02", "humidity"
+            ]]
+
+        y = self.occupancy['occupancy'].astype(int)
+
+        X = X.view((float, len(X.dtype.names)))
+        
+        with self.assertRaises(YellowbrickError):
+            visualizer = LearningCurveVisualizer(LinearSVC(), 
+                train_sizes=10000,
+                cv=ShuffleSplit(n_splits=100, test_size=0.2, random_state=0))
+            visualizer.fit(X, y)
+            visualizer.poof()
```

Comment:
I'm getting an IndexError when I run this locally.

======================================================================
ERROR: Test learning curve with bad input for training size.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/ndanielsen/ddl/jyb/tests/test_classifier/test_learning_curve.py", line 150, in test_learning_curve_bad_trainsize
    visualizer.fit(X, y)
  File "/Users/ndanielsen/ddl/jyb/yellowbrick/classifier/learning_curve.py", line 116, in fit
    self.estimator, X, y, cv=self.cv, n_jobs=self.n_jobs, train_sizes=self.train_sizes)
  File "/usr/local/var/pyenv/versions/yb-dev/lib/python3.5/site-packages/sklearn/model_selection/_validation.py", line 756, in learning_curve
    n_max_training_samples)
  File "/usr/local/var/pyenv/versions/yb-dev/lib/python3.5/site-packages/sklearn/model_selection/_validation.py", line 808, in _translate_train_sizes
    n_ticks = train_sizes_abs.shape[0]
IndexError: tuple index out of range